diff -ur rxvt/src/init.c rxvt-max/src/init.c
--- rxvt/src/init.c	2004-01-29 15:54:20.000000000 -0800
+++ rxvt-max/src/init.c	2005-06-22 05:54:16.790283200 -0700
@@ -1156,7 +1156,7 @@
  */
 
 #ifdef PREFER_24BIT
-    attributes.background_pixel = r->PixColors[Color_fg];
+    attributes.background_pixel = r->PixColors[Color_bg];
     attributes.border_pixel = r->PixColors[Color_border];
     attributes.colormap = XCMAP;
     r->TermWin.parent[0] = XCreateWindow(r->Xdisplay, Xroot,
@@ -1174,7 +1174,7 @@
 					       r->szHint.height,
 					       r->TermWin.ext_bwidth,
 					       r->PixColors[Color_border],
-					       r->PixColors[Color_fg]);
+					       r->PixColors[Color_bg]);
 #endif
     rxvt_xterm_seq(r, XTerm_title, r->h->rs[Rs_title], CHAR_ST);
     rxvt_xterm_seq(r, XTerm_iconName, r->h->rs[Rs_iconName], CHAR_ST);
